instructor/common.py

```diff
@@ -449,7 +450,7 @@ def assignments_edit_treatments(request):
         'g/L',
         'nM',
         u'\u03BCM', #uM
-        'mm',
+        'mM',
```

Comment:
Nevermind -- it's a concentration not a distance. It looks like it's millimolar.